AutolayoutPractice/RemindersList/Views/RemindersListViewController.swift

```diff
@@ -9,8 +9,8 @@ import Foundation
 import UIKit
 
 class RemindersListViewController: UIViewController, AddReminderDelegate {
-    func addReminder(title: String, description: String, dueDate: Date) {
-        viewModel.addReminder(title: title, description: description, dueDate: dueDate)
+    func addReminder(title: String, description: String, dueDate: Date, isPriority: Bool) {
```

Comment:
As you can see, this method is slowly starting to snowball. The more product requirements are added to reminder, the larger this method will be. To solve this in the short term you can wrap these in a `RemindersFormInput`, which contains all the form fields from the reminder detail path. In the long term, you should save this to a `RemindersStore` which contains the source of truth of reminder data for the app. Benefit is: 1 write to the source of truth. Anywhere in the app can read / listen for data changes